addons/scopes/stringtable.xml

```diff
@@ -4,78 +4,97 @@
         <Key ID="STR_ACE_Scopes_DisplayName">
             <English>Scopes</English>
             <Japanese>スコープ</Japanese>
+            <Polish>Celowniki optyczne</Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_enabled_displayName">
             <English>Enable ACE Scope adjustment</English>
             <Japanese>ACE スコープ調節を有効化</Japanese>
+            <Polish>Włącz ustawienia celowników optycznych ACE</Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_enabled_description">
             <English>Enable adjustmet turrets on high powered scopes</English>
             <Japanese>高倍率スコープでACE スコープ調節を有効化</Japanese>
+            <Polish>Włącz pokrętła regulacyjne </Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_forceUseOfAdjustmentTurrets_displayName">
             <English>Force adjustment turrets</English>
             <Japanese>ACE スコープ調節を有効化</Japanese>
+            <Polish>Wymus uzycie poręteł regulacyjnych</Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_forceUseOfAdjustmentTurrets_description">
             <English>Force usage of adjustmet turrets on high powered scopes</English>
             <Japanese>高倍率スコープで調整の使用を強制させます</Japanese>
+            <Polish>Wymuś użycie pokręteł regulacyjnych dla celowników o duzym powiększeniu</Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_correctZeroing_displayName">
             <English>Correct zeroing</English>
             <Japanese>ゼロイン調節</Japanese>
+            <Polish>Poprawka zerowania </Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_correctZeroing_description">
             <English>Corrects the zeroing of all small arms sights</English>
             <Japanese>全ての小口径用照準器のゼロインを調節します</Japanese>
+            <Polish>Poprawia zerowanie wszystkich celowników broni ręcznej</Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_overwriteZeroRange_displayName">
             <English>Overwrite zero distance</English>
             <Japanese>ゼロイン距離を上書き</Japanese>
+            <Polish>Nadpisuje ustawienie dla zerowego dystansu </Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_overwriteZeroRange_Description">
             <English>Uses the 'defaultZeroRange' setting to overwrite the zero range of high power scopes</English>
             <Japanese>'defaultZeroRange'設定を使う高倍率スコープのゼロイン距離を上書きします</Japanese>
+            <Polish>Używa/nadpisuje 'standardowej odległości zerowania' zamiast ustawionej odległości zerowania dla celowników o duzym przybliżeniu </Polish>
```

Comment:
This would IMHO be better description:
Używa tego ustawienia jako domyślnej odległości zerowej dla lunet celowniczych